scripts/generate-jwt-keys.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# Generate public/private JWKs for asymmetric JWTs (RSA 4096).
```

Comment:
Turns out this is somewhat redundant with the existing `generate_jwt_signing_key` management command, but there's a bit of a chicken-and-egg problem as that command still needs to be converted over to use PyJWT. That might mean dropping the nice "add to existing key set" feature. In the meantime this PR will just have to serve as reference (and a working source of keys), while I do a key rotation.